<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Calibrage</class>
 <widget class="QWidget" name="F_Calibrage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>201</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>341</width>
    <height>167</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QDoubleSpinBox" name="SBx_Valeur1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SBx_Valeur2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_Valeur1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mesurer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_Valeur2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mesurer</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Lcd_Valeur1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Lcd_Valeur2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_Valeur1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Valeur 1 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_Valeur2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Valeur 2 :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_Fermer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Bt_Valeur1</sender>
   <signal>clicked()</signal>
   <receiver>F_Calibrage</receiver>
   <slot>on_MesurerValeur1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Bt_Valeur2</sender>
   <signal>clicked()</signal>
   <receiver>F_Calibrage</receiver>
   <slot>on_MesurerValeur2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Bt_Fermer</sender>
   <signal>clicked()</signal>
   <receiver>F_Calibrage</receiver>
   <slot>on_Fermer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_MesurerValeur1()</slot>
  <slot>on_MesurerValeur2()</slot>
  <slot>on_Fermer()</slot>
 </slots>
</ui>
